ui.apps/src/main/content/jcr_root/apps/core/fd/components/formsportal/searchlister/v1/searchlister/searchlister.html

```diff
@@ -0,0 +1,46 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2021 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<sly data-sly-use.model="com.adobe.cq.forms.core.components.models.formsportal.SearchAndLister"
+     data-sly-use.logic="${'searchlister.js' @ layout=model.layout}"
+     data-sly-use.clientlib="${'/libs/granite/sightly/templates/clientlib.html'}"
+     data-sly-use.items="${logic.layoutPath}"></sly>
+<div class="cmp-searchlister" data-cmp-is="formssearch" id="${model.id}" data-queryPath="${resource.path}.model.json">
+    <h1 class="cmp-searchlister__heading" data-sly-text="${model.title}">Forms Portal</h1>
```

Comment:
What about localisation ?

Reply:
Localization will be handled at the Sling model level for these strings.
I missed adding it for model title and description (it's there for tooltip text). Will add missing I18n calls.

Reply:
Sling model should be language agnostic, all i18n related code should be part of the mark-up

Reply:
okay

Reply:
But then these won't be available as localized in the JSON?
Is that fine?

Reply:
Localisation cannot be done due to difference of localisation frameworks.
Component will show whatever content was authored as-is.